### PR TITLE
test: update space role templateref

### PR DESCRIPTION
**Pull request by xcoulon**
verify that space roles and rolebindings are updated
as expected when the templateref is changed in the
`NSTemplateSet.Spec.SpaceRoles` field

Fixes https://issues.redhat.com/browse/CRT-1339

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: test/e2e/space_test.go [hunk 1]
```diff
@@ -241,6 +242,142 @@ func TestSpaceRoles(t *testing.T) {
...
+				assert.NoError(t, err)
+				err = memberAwait.WaitUntilRoleBindingDeleted(ns, "user1-space-admin")
+				assert.NoError(t, err)
+				err = memberAwait.WaitUntilRoleBindingDeleted(ns, "user2-space-admin")
+				assert.NoError(t, err)
+
+			})
+		})
+
+		t.Run("with same content", func(t *testing.T) {
```
Comment (MatousJobanek) [inline test/e2e/space_test.go:318]: I don't think that we need this test here - it doesn't test anything extra which wouldn't be covered in the previous test. 
But I'll leave it up to you ;-)

**Review by MatousJobanek — APPROVED** [review]
looks good :+1:

File: test/e2e/space_test.go (hunk 1 above)
Comment (xcoulon) [reply test/e2e/space_test.go:318]: yeah, probably, but I'd rather keep it for now. We can remove it later if it causes too much maintenance

Comment (MatousJobanek) [reply test/e2e/space_test.go:318]: it's not only about the maintenance, but also the execution time :wink:

Comment (xcoulon) [reply test/e2e/space_test.go:318]: right :/ let me remove it, then

**Pull request merged**